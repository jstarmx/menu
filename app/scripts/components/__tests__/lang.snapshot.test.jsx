import React from 'react';
import renderer from 'react-test-renderer';

import Lang from '../lang';

it('renders correctly', () => {
  const tree = renderer.create(
    <Lang />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
