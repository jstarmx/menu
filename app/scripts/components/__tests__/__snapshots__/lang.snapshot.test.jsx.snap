// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section
  className="header__item  lang"
>
  <div
    className="flag-sprite"
    dangerouslySetInnerHTML={
      Object {
        "__html": "test-file-stub",
      }
    }
  />
  <button
    className="header__button"
    onClick={[Function]}
  >
    <svg
      className="flag"
      dangerouslySetInnerHTML={
        Object {
          "__html": "<use xlink:href=\\"#en\\"></use>",
        }
      }
    />
  </button>
</section>
`;
